db/schema.sql

```diff
@@ -13,6 +13,7 @@ CREATE TABLE "user_accounts" (
 	"user_id"	INTEGER NOT NULL,
 	"label"	TEXT NOT NULL,
 	"value"	TEXT NOT NULL,
+	"type"	INTEGER NOT NULL,
```

Comment:
I see, so user accounts is the table for the social media handles. It's a small nit but it might be a little confusing because the name "user_accounts" can also correspond to the fact that users have username/password associated with their web account for the app.

Reply:
Yeah, we were debating on a name because they may not have a 'handle' for everything like email or YouTube channel link, plus the table encompasses more than _just_ that (website or name, their handle / link / etc., **and** account type). 

Open to suggestions for a table name that is general enough to apply; since it isn't used yet in the application, it is still capable of being changed without affecting any of our current code.